db/design_docs/poi.json

```diff
@@ -0,0 +1,12 @@
+{
+   "_id": "_design/poi",
+   "language": "javascript",
+   "views": {
+       "byId": {
+           "map": "function(doc) {\n  if (doc.type === 'meta') {\n    emit(doc._id, null)\n  }\n}"
+       },
+       "currentVersionById": {
+           "map": "function(doc) {\n  if (doc.type === 'meta') {\n    emit(doc._id, {'_id': doc.current._id})\n  }\n}"
```

Comment:
there is some magic here: as the meta document emits the id of another document in the format `{'_id': doc.current._id}`, using the parameter `include_doc` will not return the emitting document but the document with the id `doc.current._id`.
This is called [Linked documents](https://wiki.apache.org/couchdb/Introduction_to_CouchDB_views#Linked_documents)